servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/pom.xml

```diff
@@ -10,9 +10,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-multiple-queue</artifactId>
+  <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-multiple</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-multiple-queue;dependency} -->
+  <version>1.0.0</version> <!-- {com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-multiple;dependency} -->
```

Comment:
you can just delete the whole lable: <!-- {com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-multiple;dependency} -->. 
Same for other pom.xml.